### PR TITLE
Update docs to reflect `kubectl run` creates deployments by default i…

### PR DESCRIPTION
**Pull request by pwittrock**
…nstead of replicatino controllers.

### CODE REVIEW

**Comment by janetkuo** [comment]
You'll need to rebase after #23305 is merged.

**Comment by pwittrock** [comment]
Thanks for the heads up

File: examples/simple-nginx.md [hunk 1]
```diff
@@ -54,16 +54,16 @@ Once the pods are created, you can list them to see what is up and running:
...
-kubectl get rc
+kubectl get deployment
 ```
 
-To delete the two replicated containers, delete the replication controller:
+To delete the two replicated containers, delete the deployment:
 
 ```bash
-kubectl delete rc my-nginx
+kubectl delete deployment my-nginx
```
Comment (janetkuo) [inline examples/simple-nginx.md]: Suggest moving this to the bottom, otherwise the below `kubectl expose` would fail.

Comment (janetkuo) [inline examples/simple-nginx.md]: of if you want to combine run and expose:
```
$ kubectl run my-nginx --image=nginx --replicas=2 --port=80 --expose --service-overrides='{ "spec": { "type": "LoadBalancer" } }'
```

(maybe to complicated for beginner users?)

File: test/images/resource-consumer/README.md [hunk 2]
```diff
@@ -48,15 +48,14 @@ Custom metrics in Prometheus format are exposed on "/metrics" endpoint.
 
 ###CURL example
 ```console
-$ kubectl run resource-consumer --image=gcr.io/google_containers/resource_consumer:beta
-$ kubectl expose rc resource-consumer --create-external-load-balancer=true --port=8080 --target-port=8080
+$ kubectl run resource-consumer --image=gcr.io/google_containers/resource_consumer:beta --expose --service-overrides='{ "spec": { "type": "LoadBalancer" } } --port 8080'
```
Comment (janetkuo) [inline test/images/resource-consumer/README.md]: the trailing `'` should be behind the last `}`

File: test/images/resource-consumer/README.md [hunk 3]
```diff
@@ -48,15 +48,14 @@ Custom metrics in Prometheus format are exposed on "/metrics" endpoint.
...
 $ kubectl get services resource-consumer
 ```
 
 There are two IPs first one is internal, second one is the external load-balanced IP. Both serving port 8080. (Use second one)
```
Comment (janetkuo) [inline test/images/resource-consumer/README.md]: Add a `.` or `:` after `There are two IPs`.

File: test/images/resource-consumer/README.md (hunk 2 above)
Comment (pwittrock) [inline test/images/resource-consumer/README.md]: Good catch.  Copy paste error

File: test/images/resource-consumer/README.md (hunk 3 above)
Comment (pwittrock) [inline test/images/resource-consumer/README.md]: Done.

File: examples/simple-nginx.md (hunk 1 above)
Comment (pwittrock) [inline examples/simple-nginx.md]: Moved.

For this simple example, I think having 2 separate commands may demonstrate the concepts better.  Don't feel strongly about it though.

**Comment by pwittrock** [comment]
Thanks Janet.  PTAL.

**Comment by janetkuo** [comment]
LGTM. Feel free to apply the tag. We may need to hand merge this.

**Comment by pwittrock** [comment]
squashed

**Comment by dchen1107** [comment]
Will manually merge once the test passed.

**Pull request merged**